Sources/Extensions/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -56,6 +56,17 @@ public extension Array {
 		guard startIndex..<endIndex ~= index else { return nil }
 		return self[index]
 	}
+    
+    /// SwifterSwift: Get the n last elements.
+    ///
+    ///        [1, 7, 1, 2, 4, 1, 6].last(n: 3) -> [4, 1, 6]
+    ///
+    /// - Parameter n: maximum number of elements.
+    /// - Returns: array slice of the n last elements, otherwise an empty array
+    public func last(n: Int) -> ArraySlice<Element> {
```

Comment:
@Vyax Thanks for your contribution 👍! I'm pretty sure this extension does the same thing as `suffix(n:)` in the Swift Standard Library.

Reply:
Thank you, I wasn't aware of that.